.gitignore

```diff
@@ -64,7 +64,6 @@ network_closure.sh
 
 # autogenerated code we don't want to check in
 ./go-bindata
-upup/models/bindata.go
```

Comment:
@justinsb @mikesplain could you please give your opinion about this change. Is it because of conflicts or why its not there?

But I do agree, it would be nice to have this in version control. We are using kops code as library and we must do dirty tricks to get it working https://github.com/ElisaOyj/kops-autoscaler-openstack/blob/master/Makefile#L31 (we basically compile bindata and upload it there and use it)

Reply:
I asked about this a while back in one of the office hours and at that time the exclusion of bindata.go was still necessary. Unfortunately I can't remember all the details of why nor can I find the video recording of it. I had asked because I also do a similar hack in my makefile.

If Justin does not reply I'll ask again in the upcoming office hours and document his answer this time.

Reply:
@zetaab @rdrgmnzs  @justinsb @mikesplain What can I do to advance this work?
Please help me, Thanks!